stores/filters.slice.ts

```diff
@@ -29,7 +29,8 @@ export const filtersSlice = createSlice({
     updateTeamSize: (state, action: PayloadAction<number>) => {
       state.teamSize = action.payload;
     },
-    updateTeamBalance: (state, action: PayloadAction<TeamBalance>) => {
+    updateTeamBalance: (state, action: PayloadAction<TeamBalance>) => {      
+      if(action.payload)
```

Comment:
I think that we should move the branching logic out of the redux slice. It would make more sense not to dispatch the action if we don’t have a valid team balance value.

Reply:
Sure